Description: A second part of patch to update AdminDocsTest to fix test suite.
Source: upstream, http://code.djangoproject.com/changeset/13764

Index: /django/branches/releases/1.2.X/tests/regressiontests/admin_views/tests.py
===================================================================
--- a/tests/regressiontests/admin_views/tests.py (revision 13750)
+++ b/tests/regressiontests/admin_views/tests.py (revision 13764)
@@ -2211,6 +2211,6 @@
 
             # An app tag exists in both the index and detail
-            self.assertContains(response, '<h3 id="flatpages-get_flatpages">get_flatpages</h3>')
-            self.assertContains(response, '<li><a href="#flatpages-get_flatpages">get_flatpages</a></li>')
+            self.assertContains(response, '<h3 id="comments-get_comment_count">get_comment_count</h3>')
+            self.assertContains(response, '<li><a href="#comments-get_comment_count">get_comment_count</a></li>')
 
             # The admin list tag group exists
